app/jobs/click_counter_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Update click counts for popular URLs for a given domain
+class ClickCounterJob < ActiveJob::Base
+  queue_as :searchgov
+
+  def perform(domain:)
+    ClickCounter.new(domain: domain).update_click_counts
+  end
```

Comment:
`ClickCounter.new(domain: domain)` will never return `nil`. It will raise an error if the required `domain` keyword argument is not included, and it returns an instance of `ClickCounter` if the domain is `nil`.

Reply:
👍